SpecializedServers/MultiSocks/DirtySocks/Messages/PersOut.cs

```diff
@@ -9,15 +9,15 @@ public class PersOut : AbstractMessage
         public string? EXticker { get; set; }
         public string? IDLE { get; set; }
         public string? LA { get; set; }
+        public string LKEY { get; set; }
```

Comment:
So, for the PERSOUT I suggest keeping the lkey in the end like this:

public string LKEY { get; set; } = "3fcf27540c92935b0a66fd3b0000283c";

Reply:
This will be addressed but for now not major issue